mlibrary/lib.umich.edu
# Website 103 rehype update

# Overview
Rehype / unified had a major version upgrade several months ago. I debated removing these plugins, but I do admit I'm still not 100% sure what they're doing. Instead, for security purposes, I've updated to the latest version and rewrote the `html.js` file to work with the latest upgrade.

> This pull request resolves [WEBSITE-103 Remove / Update Unified / Rehype](https://mlit.atlassian.net/jira/software/projects/WEBSITE/boards/27?selectedIssue=WEBSITE-103)

## Testing
- Make sure the PR is consistent in these browsers:
  - [] Safari (Josh still doesn't have Safari)
- Run accessibility tests:
- _How to use the feature_.
